node/collation-generation/src/metrics.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
2022

_I just had to_

Reply:
The source code was just copied, hence left as is - copying, in parts or full, does not alter the copyright.